src/AdminSettings.vue

```diff
@@ -206,11 +206,11 @@
 					type="text"
 					name="email_verification_hint"
 					:disabled="loading"
-					placeholder="Please create your username following the scheme 'firstname.lastname'."
+					placeholder="Welcome to Nextcloud. Please verify your email address to continue with the registration."
```

Comment:
Hmm, not sure. The email text before already says:

> ## Registration
> Email address verified, you can now complete your registration. Click the button below to continue.

Reply:
This string is not for a message sent inside the email? The username was already choosen so, the message in the email shoul not advise about the username again.

Or am I misundestaning the flow of steps of registration?

Reply:
As you can see in the gif of the readme you first enter the email and have to confirm the email code before you enter further details: https://github.com/nextcloud/registration#registration

Reply:
Thanks you very much, as you can see I did not know well the flow.

Knowing the flow, would it be possible to incorporate this modification at line 209?, because it is a message sent by the server to the client, so it could be humble to welcome the client in this string. Then the admin could change this string to another.

This evening I will install this app and check in my personal Nextcloud instance. Remember I come from the translation portal Transifex and this string only seems extrange to me. If we see I am wrong, I close this PR. =)